examples/mobile_base.py

```diff
@@ -1,31 +1,27 @@
 import gym
-import planarenvs.mobileBase
+import planarenvs.mobile_base  # pylint: disable=unused-import
 import numpy as np
 
 obstacles = False
 
 
 def main():
-    env = gym.make("mobile-base-acc-v0", render=True, dt=0.01)
-    defaultAction = [0.4]
-    n_episodes = 1
+    env = gym.make("mobile-base-vel-v0", render=True, dt=0.01)
+    action = [0.4]
     n_steps = 1000
-    cumReward = 0.0
-    for e in range(n_episodes):
-        ob = env.reset(pos=np.array([-2.0]), vel=np.array([0.5]))
-        if obstacles:
-            from examples.obstacles import sphereObst1, sphereObst2
+    ob = env.reset(pos=np.array([-2.0]), vel=np.array([0.5]))
+    if obstacles:
+        from examples.obstacles import (
```

Comment:
Rather a personal question: Is it best practice to only import modules, if necessary? I usuallly import everything on top of each file, no matter if it is used or not.

Reply:
So generally, you always want to import at the top (it is even a listing error if you don't do that).
However, in this case, the obstacles require an optional dependency. So if the dependency is not met, the example should still function without the obstacles, thus the import must be conditional.